earth_enterprise/SConstruct

```diff
@@ -134,21 +119,54 @@ env = khEnvironment(ENV={'PATH' : '/bin:/bin:/usr/bin'},
                     exportdirs=exportdirs,
                     installdirs=installdirs,
                     JAVA_HOME="/opt/google/java")
-env.Alias('install', installdirs['root'])
+
+def getArgumentVarsAsString():
+  vars=''
+  for var, value in ARGUMENTS.iteritems():
+    if var != 'installdir':
+      vars += var + '=' + str(value) + ' '
+  return vars[:-1]  # Remove the trailing space
+
+arg_vars = getArgumentVarsAsString()
+num_jobs = GetOption('num_jobs')
+
+# Build Fusion and Server
+build_cmds = {'build': [
+    'cd src; scons -j%d %s third_party' % (num_jobs, arg_vars),
+    'cd src; scons -j%d %s' % (num_jobs, arg_vars)
+]}
+run_build = env.PhonyTargets(**build_cmds)
```

Comment:
I timed this and it is actually just under 5 seconds on my machine.  Given that the only reason we need to call the third_party build anyway is because of the skia library (see #176), I'm going to suggest waiting until the skia build is fixed and then removing the extra call as part of that ticket.  What do you think?